data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSource.java

```diff
@@ -80,6 +80,9 @@ public void start(final Buffer<Record<Log>> buffer) {
         }
         if (server == null) {
             final ServerBuilder sb = Server.builder();
+
+            sb.disableServerHeader();
```

Comment:
Is this related to log4j CVE ?

Reply:
No, this is not related to log4j. It was added as an additional security protection to prevent unauthorized persons from know what type of server is in use.